apps/balancer-tools/src/app/apr/round/(components)/PoolsCards.tsx

```diff
@@ -1,13 +1,19 @@
 import { ChevronRightIcon } from "@radix-ui/react-icons";
 
-import { DunePoolData } from "#/lib/dune";
 import { fetcher } from "#/utils/fetcher";
 import { formatNumber } from "#/utils/formatNumber";
 
+export interface PoolData {
+  pct_votes: number;
+  symbol: string;
+  votes: number;
+  apr: number;
```

Comment:
why did we need to define this here again?

Reply:
I think placing this here will make it easier when displaying data on the rounds page